Mathlib/RepresentationTheory/FDRep.lean

```diff
@@ -186,3 +187,41 @@ theorem dualTensorIsoLinHom_hom_hom : (dualTensorIsoLinHom ρV W).hom.hom = dual
   rfl
 
 end FDRep
+
+noncomputable section equivFullSubcategory
+
+namespace FDRep
+
+variable {k G : Type u} [Field k] [Monoid G] {V W : FDRep k G}
+
+/- TODO: Unify how to phrase lemmas/theorems - Use V or (forget₂ ... V) or V.ρ.asModule? -/
```

Comment:
I guess this still needs cleaning up.

Reply:
Hmm, I don't plan to deal with the TODO in this PR. For now I use `forget\2 ... V`, since it seems a lot of lemmas are stated so.

Reply:
Is it okay for me to merge this PR? Maybe the TODO can be handled some other time.